README.md

```diff
@@ -102,12 +102,13 @@ Starting with v3.1 you can now use different ways of configuring lint-staged:
   - `.lintstagedrc.json`
   - `.lintstagedrc.yaml`
   - `.lintstagedrc.yml`
-- `lint-staged.config.js`, `.lintstagedrc.js`, or `.lintstagedrc.cjs` file in JS format
+- `.lintstagedrc.mjs` or `lint-staged.config.mjs` file in ESM format
```

Comment:
Perhaps it is worth mentioning that the default export is expected for mjs files

Reply:
`export default` value for ESM and `module.exports` value for CommonJS; I'll add that in, good suggestion.

Reply:
Done.